x-pack/test/functional/services/ml/data_visualizer_table.ts

```diff
@@ -52,6 +53,16 @@ export function MachineLearningDataVisualizerTableProvider({ getService }: FtrPr
       return rows;
     }
 
+    public async assertTableRowCount(expectedRowCount: number) {
+      await retry.tryForTime(5000, async () => {
+        const filteredRows = await this.parseDataVisualizerTable();
+        expect(filteredRows).to.have.length(
+          expectedRowCount,
+          `Filtered Data Visualizer table should have ${expectedRowCount} row(s) (got '${filteredRows.length}')`
```

Comment:
```suggestion
          `Data Visualizer table should have ${expectedRowCount} row(s) (got '${tableRows.length}')`
```
When this method is called, the table is not necessarily filtered, so I would suggest to remove `filtered` from the variable name and message wording.

Reply:
I can see that you've renamed the variable, but the message still says `Filtered Data Visualizer table should [...]`. Nothing critical, so I'm ok with it being fixed in a future PR if you want to get the PR in quickly.

Reply:
Fixed here [`5581931` (#86071)](https://github.com/elastic/kibana/pull/86071/commits/55819316abe5e29c01932dbc71b53f51ece2f9d6)